src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-100301.mdx

```diff
@@ -0,0 +1,27 @@
+---
+subject: 'Python agent'
+releaseDate: '2024-11-25'
+version: 10.3.1
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: []
+bugs: ['Fix package version uploading logic']
+security: []
+---
+
+## Notes
+
+This release of the Python agent fixes a bug in the package version collector that disabled telemetry from being sent to New Relic in certain cases.
+ 
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+    
+## Bug fixes
+
+* Fix package version uploading logic
+
+  * Previously, if the package's versioning mechanism was a class property, the function itself would get sent into the payload, so the environment stats would show the `__repr__()` of the versioning class. Once this task was tweaked and moved to the harvest cycle in v10.3.0, this resulted in a failed harvest cycle, which effectively disabled any telemetry from being sent. This release adds logic to prevent anything that's not a valid data type from being sent within the module payload.
```

Comment:
```suggestion
  * Previously, if the package's versioning mechanism was a class property, the function itself would get sent into the payload, so the environment stats would show the `__repr__()` of the versioning class. Once this task was tweaked and moved to the harvest cycle in v10.3.0, this resulted in a failed harvest cycle when the version was converted to JSON, which effectively disabled any telemetry from being sent. This release adds logic to prevent anything that's not a valid data type from being sent within the module payload.
```